tsconfig.json

```diff
@@ -23,5 +23,8 @@
   "include": [
     "src/**/*.tsx",
     "src/**/*.ts"
+  ],
+  "exclude": [
```

Comment:
Bundle is built using the Index.tsx file, and therefore, it's only importing what it wants to export. Therefore, this exclude is not required.